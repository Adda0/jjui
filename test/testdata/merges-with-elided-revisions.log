@      yz;yzrmnvpl;.;true;false;true;true;false;some@email.com;19 hours ago;a;a;
├─┬─╮
│ │ ○  yr;yrxzkkxk;.;false;false;false;false;false;some@email.com;20 hours ago;b;b;second
│ ◆ │  p;pvsukzur;main;false;true;false;false;false;some@email.com;20 hours ago;c;c;add hello file
│ ~ │  (elided revisions)
│ ├─╯
○ │  l;lkwqqytz;.;false;false;false;false;false;some@email.com;19 hours ago;d;d;add `why` file
○ │  o;oqwrqwrm;feat;false;false;false;false;false;some@email.com;20 hours ago;f;f;add more file
○ │  x;xplxzxmu;.;false;false;false;false;false;some@email.com;20 hours ago;g;g;add another file
○ │  s;suumnpyy;.;false;false;false;false;false;some@email.com;20 hours ago;h;h;add one more file
○ │  w;wrulxpqu;.;false;false;false;false;false;some@email.com;20 hours ago;i;i;change 1
○ │  m;mnvkkxpp;.;false;false;false;false;false;some@email.com;20 hours ago;j;j;third
├─╯
◆  z;zzzzzzzz;.;false;true;false;true;55 years ago
