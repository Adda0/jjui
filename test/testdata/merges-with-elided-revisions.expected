@      yzrmnvpl
├─┬─╮  (empty) (no description set)
│ │ ○  yrxzkkxk
│ │ │  second
│ ◆ │  pvsukzur
│ │ │  add hello file
│ ~ │  (elided revisions)
│ ├─╯
○ │  lkwqqytz
│ │  add `why` file
○ │  oqwrqwrm
│ │  add more file
○ │  xplxzxmu
│ │  add another file
○ │  suumnpyy
│ │  add one more file
○ │  wrulxpqu
│ │  change 1
○ │  mnvkkxpp
├─╯  third
◆  zzzzzzzz root()